modules/hal_nordic/nrfx/nrfx_glue.h

```diff
@@ -334,6 +334,24 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
 				   NRFX_PPI_GROUPS_USED_BY_802154_DRV | \
 				   NRFX_PPI_GROUPS_USED_BY_MPSL)
 
+#if defined(CONFIG_SOC_SERIES_NRF54HX)
+#define GPIOTE_CH_OWNED_CH(node_id, prop, idx) | BIT(DT_PROP_BY_IDX(node_id, prop, idx))
+
+/* Macro for getting mask of channels assigned to the given core. It shall be used only for GPIOTE
+ * instances which has split ownership.
+ */
+#define GPIOTE_OWNED_CHANNELS(idx)                                                                \
+	(COND_CODE_1(DT_NODE_HAS_PROP(DT_NODELABEL(gpiote##idx), owned_channels),                 \
+	((0 DT_FOREACH_PROP_ELEM(DT_NODELABEL(gpiote##idx), owned_channels, GPIOTE_CH_OWNED_CH))),\
+	(0)))
+
+#if DT_NODE_EXISTS(DT_NODELABEL(gpiote130))
+/** @brief Bitmask that defines GPIOTE130 channels reserved for use outside of the nrfx library. */
+#define NRFX_GPIOTE130_CHANNELS_USED ~GPIOTE_OWNED_CHANNELS(130)
```

Comment:
`DT_NODE_EXISTS()` is actually redundant, as `DT_NODE_HAS_PROP()` correctly handles non-existing nodes.